### ISSUE TITLE
Native Views throws 'XamlParseException'

### ISSUE THREAD

**Issue by mykyta-bondarenko-gl** [xamarin/Xamarin.Forms#4505]
### Description
```
Xamarin.Forms.Xaml.XamlParseException: Position 51:139. No static member found for ios:UIColor.Red
```
### Steps to Reproduce

1. Download a [sample](https://github.com/xamarin/xamarin-forms-samples/tree/master/Forms2Native) 
2. Build it
3. Run it on a device 

### Expected Behavior
It runs successfully 

### Actual Behavior
It crashes on launch

### Basic Information

- VSfM
https://gist.github.com/mykyta-bondarenko-gl/a25f01c1f177999604edacab8b15310d
- Output
https://gist.github.com/mykyta-bondarenko-gl/b7508755df1322460d8d18ab8519377b
- Xamarin.Forms
3.4.0.1008975
- Device 
iPhone 7, iOS 12.1 (16B92)

### Reproduction Link

[Sample](https://github.com/xamarin/xamarin-forms-samples/tree/master/Forms2Native)

**Comment by davidbritch** [comment]
Sample works fine in the simulator, for both debug and release builds. Just fails to work when you run it on device.

**Comment by StephaneDelcroix** [comment]
@davidbritch is it due to linking ? does having a reference in code to the property "fixes" it ?

```csharp
if (false)
{
  var color = UIColor.Red;
}
```

**Comment by davidbritch** [comment]
Yes, it's a linker issue. Switching off the linker made the sample work on a device, as did referencing every color used by the app.

Feel free to close this here, and I'll fix the affected samples tomorrow (already spoken to Mykyta).
